.travis.yml

```diff
@@ -0,0 +1,42 @@
+notifications:
+  slack:
+    secure: cuB9waMrZiRnJjJLq68NTSg1nB+ypbCTy4cphu0VjfuDYhmQc4MgnkvmzLntsE9X9Oj8kSP51rRjD+IJJ8qT1GTMi/5XAqKNzncONEdx5T0Tict0xxAaY8OcumOV68tisQ1jEeAVA65zcvPoyWOVv+ha4MdHLx8+J85h/zOFMRALf2tJ8tUnC2iHidWCYsQS8vQEkKp1eDWJRdcJY+OrmeV5tlgWXP56b6pRdnhoWwOkwtRols6yoPw1j6uNewb3N2uvJIIw5g7ji209nXFM+/HFGLLqNlWUpqickbaZEG/m5VqeuxeaZTl6jlW2w2+B5zyMPKR8NHeSQ8+c6Si08LbgQDGkLh1xHNQOTxqTEOHUOsLgBm7Y+NX1QDfIfclvY+dhWhUiNHVT39+om/I/8MvpuDLLtujijJIq+mluJdjuAkWWXCVqYGMll0SalvEQf43Os6yxZnewJ0QjlajUY8txgPR9yyVLQduddiMymSIwAAKE3gLlhTpl59W3kRcKcd0tTUtnqH7KAg6GBnFqkqnUPv9Fmws+eOvyThg36gX7+RWbo/mwQZCH265qY6cf02pbsU9Hoo5HTXd7nWi4G/NL3vKR3yySrOCzfzgwY2vozwrWY09pKphNtRhtLZkXEKdvDyFt9i6/nh6muiaZEQ22dxxK/1iBKTeR+IW5p48=
+language: python
+python: "2.7"
+addons:
+  apt:
+    packages:
+        - build-essential
+        - python-dev
+        - git
+        - mercurial
+        - python-pip
+        - libopenmpi-dev
+        - openmpi-bin
+        - libblas-dev
+        - liblapack-dev
+        - gfortran
+os:
+  - linux
+
+env:
+  global:
+    - CC=mpicc
+
+before_install:
+  - pip install -U pip
+  - pip install -U virtualenv
+  - pip install flake8
+
+install:
+  - curl -O https://raw.githubusercontent.com/firedrakeproject/firedrake/master/scripts/firedrake-install
+  - python ./firedrake-install --disable-ssh --minimal-petsc
+  - . ./firedrake/bin/activate
+  - python setup.py install
+
+script:
+  - make lint
+  - make test
+
```

Comment:
Noted. Isn't it ironic that the script checking linting errors isn't checked for this itself...